frontend/src/metabase/ui/components/buttons/Button/Button.styled.tsx

```diff
@@ -158,13 +159,13 @@ export const getButtonOverrides = (): MantineThemeOverride["components"] => ({
 });
 
 const getPrimaryColor = (theme: MantineTheme, colorName: string) => {
-  return theme.fn.themeColor(colorName, theme.fn.primaryShade());
+  return theme.fn.themeColor(colorName);
 };
 
-const getHoverColor = (theme: MantineTheme, primaryColor: string) => {
-  return theme.fn.rgba(primaryColor, 0.88);
+const getHoverColor = (primaryColor: string) => {
+  return Color(primaryColor).alpha(0.88).string();
```

Comment:
It's a bummer that `theme.fn.rgba` [doesn't expect anything outside a HEX format.](https://github.com/mantinedev/mantine/blob/v6/src/mantine-styles/src/theme/functions/fns/rgba/rgba.test.ts#L4), so I use `color` that is already working.